docs/qute/TemplateSupport.md

```diff
@@ -111,3 +111,27 @@ Qute support displays the Java type of the parameter names as an inlay hint.
 ![Inlay Hints for Qute](images/QuteTemplateInlayHint.gif)
 
 All inlay hints can be turned off by disabling `qute.inlayHints.enabled` in settings. To remove inlay hints on section parameter types only, disable `qute.inlayHints.showSectionParameterType` in settings.
+
+## Surround with Commands (Wrap)
+
+This command provides the ability to select Qute content and surround it some content. To execute this command you can use the command palette (`Ctrl+Shift+P`) and type `Surround`.
+
+If you prefer using keyboard to process the command, you need to associate this command with a keybinding. See [Keyboard Shortcuts editor](https://code.visualstudio.com/docs/getstarted/keybindings#_keyboard-shortcuts-editor) for more informations.
+
+### Surround with Section
+
+Surrounds the selected content with a section and completion will be triggered. You can also do an empty section on the section you would like to surround and the correct section will selected.
+
+![Qute Surround with Section](images/QuteTemplateSurroundWithSection.gif)
```

Comment:
You don't apply #each in the demo, it is because you are waiting for my PR?

Reply:
I will update the demo :+1: